PostHog/Classes/PHGPostHogConfiguration.h

```diff
@@ -123,6 +123,13 @@ typedef NSMutableURLRequest *_Nonnull (^PHGRequestFactory)(NSURL *_Nonnull);
  */
 @property (nonatomic, assign) BOOL shouldSendDeviceID;
 
+/**
+ * Whether the posthog client should load feature flags when initialized.
+ * Changing the value of this property after initializing the client will have no effect.
```

Comment:
We can add a comment that they can be reloaded by calling the `reloadFeatureFlags` method.

Reply:
Updated.